capellambse/model/diagram.py

```diff
@@ -303,37 +318,42 @@ def type(self) -> modeltypes.DiagramType:
             return modeltypes.DiagramType.UNKNOWN
 
     @property
-    def activated_filters(self) -> list[str]:
-        """Return a sequence of activated filters"""
-        return aird.parser._filters.get_filters_from_diagram(
-            self._model._loader, self._element.uid
-        )
+    def filters(self) -> aird.parser._filters.ActiveFilters:
```

Comment:
I'm not happy with the return annotation here. First, it can be simplified as `aird.ActiveFilters`, but IMO we can easily consider this entire class to be an implementation detail and just announce it as `cabc.MutableSet[str]`. And then we don't even need to expose the `ActiveFilters` class publicly in the first place.